helpers/currency-format.js

```diff
@@ -22,10 +23,24 @@ const formatCurrency = (amount, options) => {
   });
 
   if (options.isMoney) {
-    return new Intl.NumberFormat(options.locale, {
+    // Set the amount of decimals we want to show, it's 2 or 0
+    const hasTrailingZeroZero = amount === 0 || amount % 1 === 0;
```

Comment:
I wrote this part wrong myself, hehe. `amount % 1 = 0` a.k.a. `0 % 1 = 0`, so checking `amount === 0` is redundant.